src/VisualStudio/Core/Def/Implementation/Workspace/SourceGeneratedFileManager.cs

```diff
@@ -300,6 +274,8 @@ public OpenSourceGeneratedFile(SourceGeneratedFileManager fileManager, ITextBuff
                     UpdateBufferContentsAsync,
                     asyncListener: _fileManager._listener,
                     _cancellationTokenSource.Token);
+
+                _workspace.OnSourceGeneratedDocumentOpened(documentIdentity, textBuffer.AsTextContainer());
```

Comment:
i don't love that this is done during construction of an object.  feels weird since the file itself may not be in any expected collections that the manager owns.  should this move to another location?

Reply:
It moved as a part of other fixes.